standard/lua.lua

```diff
@@ -0,0 +1,60 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Lua
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = {}
+
+function Lua.moduleExists(name)
+	if package.loaded[name] then
+		return true
+	else
+		for _, searcher in ipairs(package.searchers or package.loaders) do
+			local loader = searcher(name)
+			if type(loader) == 'function' then
+				-- luacheck: ignore

```

Comment:
What are we ignoring here?

Reply:
![image](https://user-images.githubusercontent.com/85348434/134002144-a49707fd-ecd3-4b59-8822-a85d631332ba.png)

Reply:
pacakge.preload is supposed to be read/write according to https://luacheck.readthedocs.io/en/stable/warnings.html

not sure how to configure it as such so just added the inline suppress

Reply:
Alright that seems okay. Could you add that to the ignore comment?